Plugin/CMakeLists.txt

```diff
@@ -1,17 +1,32 @@
 cmake_minimum_required(VERSION 2.8.12)
-project(NodOSVRPlugin)
+project(org_osvr_nod)
 
 find_package(osvr REQUIRED)
 
-osvr_convert_json(NodOSVR
-    NodOSVR.json
-    "${CMAKE_CURRENT_BINARY_DIR}/NodOSVR.h")
+osvr_convert_json(org_osvr_nod
+    org_osvr_nod.json
+    "${CMAKE_CURRENT_BINARY_DIR}/org_osvr_nod.h")
 include_directories("${CMAKE_CURRENT_BINARY_DIR}")
 
-osvr_add_plugin(NAME NodOSVRPlugin
+osvr_add_plugin(NAME org_osvr_nod
     CPP 
     SOURCES
-    NodOSVRPlugin.cpp
-    "${CMAKE_CURRENT_BINARY_DIR}/NodOSVR.h"
-	Settings.h
-	NodPlugin.h)
+    org_osvr_nod.cpp
+    "${CMAKE_CURRENT_BINARY_DIR}/org_osvr_nod.h")
+
+#check for openspatial dependencies and link in, fail if not found
+if(CMAKE_GENERATOR MATCHES "Win64")
```

Comment:
Also, if you look at my commit message, I specifically mention that it will only work if Nod is installed in its default location. A find function is coming soon.